### DISCUSSION TITLE
how to smoothly transition between expression

### DISCUSSION THREAD

**Discussion by Ppint** [pixiv/three-vrm#1322] [Help]
how to smoothly transition between expression?

for example

i want to change aa from 0.5 to 1 and then back to 0.5 again how to make that look smooth

**Comment by 0b5vr** [comment]
There are three ways I can come up with:

- Use animation clip. You can refer to the animation example we have in the three-vrm repository.
  - Working example: https://pixiv.github.io/three-vrm/packages/three-vrm/examples/animations.html
  - Source code: https://github.com/pixiv/three-vrm/blob/dev/packages/three-vrm/examples/animations.html
- Use an animation library like [Tween.js](https://github.com/tweenjs/tween.js)
- Implement an animation logic using math

**Reply by Ppint** [reply]
so i tried using animaton clip. everything work fine until I want to use aa ee ih ou oh in animation clip. it just stop working
